sys/net/application_layer/gcoap/forward_proxy.c

```diff
@@ -102,7 +102,7 @@ static ssize_t _forward_proxy_handler(coap_pkt_t *pdu, uint8_t *buf,
                                       size_t len, coap_request_ctx_t *ctx)
 {
     int pdu_len;
-    sock_udp_ep_t *remote = coap_request_ctx_get_context(ctx);
+    sock_udp_ep_t *remote = coap_request_ctx_get_remote_udp(ctx);
```

Comment:
```suggestion
const sock_udp_ep_t *remote = coap_request_ctx_get_remote_udp(ctx);
```

But simply adding this single `const` is not enough. It propagates to `forward_proxy.c`. Adding the `const` in `forward_proxy.c` should not be a problem.